app/Console/Commands/SendNotificationForOpenCFPs.php

```diff
@@ -31,10 +31,16 @@ class SendNotificationForOpenCFPs extends Command
      */
     public function handle()
     {
-        Conference::approved()->notShared()->openCFP()->each(function ($conference) {
-            $conference->update(['is_shared' => true]);
+        $conferences = Conference::approved()->notShared()->openCFP()->get();
+
+        if ($conferences->isEmpty()) {
+            return;
+        }
 
-            Notification::send(User::wantsNotifications()->get(), new CFPIsOpen($conference));
+        $conferences->each(function ($conference) {
+            $conference->update(['is_shared' => true]);
```

Comment:
This is a minor nitpick, but could we make this a single update?

```php
Conference::approved()->notShared()->openCFP()->update(['is_shared' => true]);
```

I know it's duplicating the query, but this way it's a single SQL call, which I like. What do you think?

Reply:
That's a good idea.  I updated it so it is now a single update query.  Rather than duplicating the whole query, I hid the common part of the query behind a `conferencesToShare` private method.